ImageInLib/ImageInLib/src/shape_registration.c

```diff
@@ -88,6 +88,10 @@ void run_registration(dataType **fixedData, dataType **movingData, dataType **re
 	{
 		finalResults = registrationStochastic3D(fixedData, movingData, finalResults, step_size, tol, zDim, xDim, yDim, movingCentroid, params);
 	}
+	else if (gdescentMethod == BLOCK_COORDINATE_DESCENT)
+	{
+		finalResults = regCoorDescent3D(fixedData, movingData, finalResults, step_size, tol, zDim, xDim, yDim, movingCentroid, params);
```

Comment:
Please try not to use abbreviations in names.
The names will be easier readable.
So in this case instead of regCoorDescent3D -> registrationCoordinateDescent3D

Reply:
Sure, will push an update to correct this